--- expected
+++ actual
@@ -8,15 +8,20 @@
 __webpack_require__.r(__webpack_exports__);
 /* harmony import */var _lib__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./lib */"./lib.js");
 
-_lib__WEBPACK_IMPORTED_MODULE_0__.answer;
+_lib__WEBPACK_IMPORTED_MODULE_0__.a;
 }),
 "./lib.js": (function (__unused_webpack_module, __webpack_exports__, __webpack_require__) {
 "use strict";
 __webpack_require__.r(__webpack_exports__);
+__webpack_require__.d(__webpack_exports__, {
+  a: function() { return a; }
+});
 
- function useModel() {
-    /* "./answer" unused */null;
+function useModel() {
+    answer;
 }
+const a = 3;
+
 }),
 
 },function(__webpack_require__) {
